test/modules/packages/collections/CollectionBulk.skipif

```diff
@@ -1,2 +0,0 @@
-# 'fifo' tasking layer results in a double free in the runtime of a pthread_condvar. Works on Qthread
```

Comment:
Is this skipif removal intentional?

Reply:
Yeah, I forgot to remove it last time. #7136 fixed the issue for the 'fifo' tasking layer, and I just remembered now. I suppose this kind of widens the topic of this PR.

Reply:
I'm fine with it being removed here, just wanted to make sure it was intentional since I didn't see it noted in the PR message